internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -45,6 +46,10 @@ const (
 	maxIndexNameLength = 63
 )
 
+// specialCharacters are potential problematic characters of pg table name
+// that are replaced with `_`.
+var specialCharacters = regexp.MustCompile("[^a-z][^a-z0-9_]*")
```

Comment:
This replaces `+-/*<>=~!@#%^&|`?()[],;:.` with `__`. The first `[^a-z]` matches `+` and `[^a-z0-9_]*` matches the rest.
I can't figure out how to produce the same number of characters. But this is fine for mangling pg table name.  Please suggest if you have better approach :hugs: